sys-utils/dmesg.c

```diff
@@ -1448,7 +1448,10 @@ int main(int argc, char *argv[])
 
 	static const ul_excl_t excl[] = {	/* rows and cols in ASCII order */
 		{ 'C','D','E','c','n','r' },	/* clear,off,on,read-clear,level,raw*/
+		{ 'F','S' },			/* file,syslog */
+		{ 'H','T','e','t',OPT_TIME_FORMAT }, /* time formats */
```

Comment:
This seems incorrect; for example, `dmesg -H --time-format ctime` forks as expected. (Unfortunately, it depends on the order of options.)

In the mailing list, we have a patch to support multiple time formats for each message and I hope it will fix the options order issue.

Reply:
I'll drop this patch for now then.
( [9ac309d](https://github.com/util-linux/util-linux/pull/2634/commits/9ac309dd5b54bce4e8a7cc39651e67f907be378d) )

If we are fine with having ordered options it's also easier validate the syslog parsing.